### PR TITLE
Getting Started - Learn how Free Code Camp Works translated into Braz…

### PR DESCRIPTION
**Pull request by samanthakem**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [x] Add new translation (feature adding new translations)
#### Checklist:

- [ ] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX
#### Description

I did the translation of the session - Getting Started **[Learn How Free Code Camp Works]** - into Brazilian Portuguese.

### CODE REVIEW

**Comment by atjonathan** [comment]
please get someone in the brazilian portuguese chat rooms to peer review this, thanks :+1:

**Comment by samanthakem** [comment]
@atjonathan I already sent the PR link to the pt-br channel ... waiting anyone get back to me :/

File: seed/challenges/00-getting-started/getting-started.json [hunk 1]
```diff
@@ -393,6 +393,83 @@
...
+          "description": [
+            [
+              "//i.imgur.com/6ibIavQ.jpg",
+              "Imagem dos 4 benefícios do Free Code Camp: Conecte-se, Aprenda Javascript, Construa seu Portfolio, Ajude Organizações Sem Fins Lucrativos",
+              "Bem-vindo ao Free Code Camp. Nós somos uma comunidade Open Source cujo objetivo é ajudá-lo(a) a aprender a programar e praticar através da construção de projetos para organizações sem fins lucrativos.",
+              ""
+            ],
+            [
+              "//i.imgur.com/Elb3dfj.jpg",
+              "Um screenshot de alguns dos nossos campistas programando Toronto.",
```
Comment (sirluis) [inline seed/challenges/00-getting-started/getting-started.json]: Uma pequena correção:
`"Um screenshot de alguns dos nossos campistas programando **em** Toronto.",`

File: seed/challenges/00-getting-started/getting-started.json [hunk 2]
```diff
@@ -393,6 +393,83 @@
...
+            ],
+            [
+              "//i.imgur.com/vLNso6h.jpg",
+              "Uma ilustração mostrando que você aprenderá HTML5, CSS3, JavaScript, Banco de Dados, Git, Node.js, React e D3.",
+              "Nós temos centenas de desafios opcionais de programação que o(a) ensinarão as principais tecnologias de desenvolvimento web como HTML5, Node.js e banco de dados.",
+              ""
+            ],
+            [
+              "//i.imgur.com/UVB9hxp.jpg",
+              "Uma imagem de um campista codando e bebendo café no Free Code Camp.",
```
Comment (sirluis) [inline seed/challenges/00-getting-started/getting-started.json]: >  "An image of a camper at a cafe building projects on Free Code Camp."
Não seria melhor algo como: Imagem de um campista numa cafeteria construindo projetos no Free Code Camp.

Comment (samanthakem) [inline seed/challenges/00-getting-started/getting-started.json]: Definitivamente sim! Obrigada.

File: seed/challenges/00-getting-started/getting-started.json [hunk 3]
```diff
@@ -393,6 +393,83 @@
...
+              ""
+            ],
+            [
+              "//i.imgur.com/pbW7K5S.jpg",
+              "Uma imagem mostrando nossos certificados de front-end, back-end e visualização de dados (400 horas cada), nossos projetos sem fins lucrativos (800 horas) e uma preparação para entrevista (80 horas) para um total de 2.080 horas de experiência em programação.",
+              "Nosso programa de estudos está dividido em 4 certificações. Estas certificações são padronizadas e instataneamente verificadas por seus clientes freelance e futuros empregadores. Assim como tudo no Free Code Camp, essas certificações são gratuitas. Nós recomendamos que as mesmas sejam feitas em ordem mas você está livre para explorar ao seu modo. As primeiras três certificações levam 400 horas cada e a cetificação final leva 800 horas e envolve a construção de projetos reais para organizações sem fins lucrativos.",
```
Comment (sirluis) [inline seed/challenges/00-getting-started/getting-started.json]: > ...são padronizadas e instataneamente verificadas...

typo fix: instantaneamente

File: seed/challenges/00-getting-started/getting-started.json [hunk 4]
```diff
@@ -393,6 +393,83 @@
...
+            [
+              "//i.imgur.com/yXyxbDd.jpg",
+              "Uma foto de nosso diretório de projetos sem fins lucrativos.",
+              "Após você completar todos os três certificados, você formará uma equipe com outro campista e usará metodologias ágeis de desenvolvimento de software para construir dois projetos reais para organizações sem fins lucrativos. Você também adicionará funcionalidades a dois antigos projetos sem fins lucrativos. Quando você terminar, você terá um portfolio com projetos reais que pessoas usam todos os dias.",
+              ""
+            ],
+            [
+              "//i.imgur.com/PDGQ9ZM.jpg",
+              "Uma imagem de campistas construindo projetos em Seoul.",
+              "Se você completar todas as 2.080 horas válidas de desafios e projetos, você ganhará nosso certificado de desenvolvimento Full Stack. Nós o(a) oferecemos prática gratuita para entrevistas de programação. Nós oferecemos ainda um espaço de trabalho onde empregadores espeficamente contratam campistas que conseguiram certificações no Free Code Camp.",
```
Comment (sirluis) [inline seed/challenges/00-getting-started/getting-started.json]: typo fix 
"espeficamente"

**Comment by Bouncey** [comment]
@samanthakem is there any progress with this PR?

**Review by ghost — CHANGES_REQUESTED** [review]
Make necessary changes commented and fix merge conflicts ❗

**Comment by atjonathan** [comment]
Closing as stale. Please reopen when you've made the changes @samanthakem 😄

**Comment by samanthakem** [comment]
I don't having permissions to re-open this issue but you will find the new PR here:

https://github.com/FreeCodeCamp/FreeCodeCamp/pull/10991

**Pull request closed without merging**